src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -228,26 +205,27 @@ CHIP_ERROR OTARequestor::SetupCASESessionManager(chip::FabricIndex fabricIndex)
     // CSM has not been setup so create a new instance of it
     if (mCASESessionManager == nullptr)
```

Comment:
After https://github.com/project-chip/connectedhomeip/pull/12148. We need to also provide the `CASEClient` and `OperationalDeviceProxy` pool to the `CASESessionManager`. The pool can be located in the server instance.

Reply:
@gjc13 Yes, I've made the changes now to pass in `CASEClient` and `OperationalDeviceProxy`